test/services/dashd.unit.js

```diff
@@ -38,6 +38,11 @@ describe('Dash Service', function() {
     }
   };
 
+  var RPC_IN_WARMUP_ERROR = new Error('Verifying blocks...');
```

Comment:
Doesn't look like an error. You probably need to catch warmup error and output this message using logger.

Reply:
It must be an `Error` instance, otherwise all of the tests (and any code that depends on it being an `Error` instance that already exists in the wild) would need to be rewritten and have more branching logic.

Since JS doesn't distinguish between warnings and errors, etc, this should be an error. The logic that checks for its use can make the distinction.

Reply:
Ok, but the message should still describe an error. Now it looks like a status report label.

Reply:
@shumkov this isn't my thing. That's the message as it currently exists. I'm fixing the bug, not changing the messages or behavior.

If you want to define new messages and differentiate between error vs status, that probably comes from the C code.

Reply:
Oh, right.